com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -3746,9 +3746,20 @@ private void ReplaceFile ()
         private string SaveMetafile()
         {
             var tempMetafilePath = Path.GetTempFileName();
-                
-            // check if path is relative or absolute
-            var fbxPath = AssetDatabase.LoadAssetAtPath(m_lastFilePath, typeof(Object)) != null ? m_lastFilePath : "Assets" + m_lastFilePath.Substring(Application.dataPath.Length);
+            
+            // Try as a relative path
```

Comment:
I think you got relative and absolute backwards (this is absolute, the next try is for a relative path)
Same goes for `ReplaceMetafile`